apps/sms/views/conversation/js/conversation.js

```diff
@@ -1927,6 +1940,18 @@ var ConversationView = {
     new OptionMenu(params).show();
   },
 
+  addRecipients: function() {
+    var draft = new Draft({
+      recipients: this.activeThread.participants,
+      content: Compose.getContent(),
+      subject: Compose.getSubject(),
+      type: Compose.getType()
+    });
+    this.initiateNewMessage({ draft });
+    // As we are creating a new draft, we can discard current Composer
+    Compose.clear();
```

Comment:
That's how I prevent `beforeLeave` to save a draft. Imo it makes sense, we are transferring the current composer to a draft, so we can clear it now. WDYT?